examples/webbilling-demo/src/tests/helpers/integration-test.ts

```diff
@@ -5,7 +5,43 @@ import { getEmailFromUserId } from "./test-helpers";
 export const ALLOW_PAYWALLS_TESTS =
   process.env.VITE_ALLOW_PAYWALLS_TESTS === "true";
 
-export const ALLOW_TAXES_TESTS = process.env.VITE_ALLOW_TAXES_TESTS === "true";
+export const SKIP_TAXES_TESTS = (() => {
+  const skipUntilDate = process.env.VITE_SKIP_TAXES_TESTS_UNTIL;
+  if (!skipUntilDate) return false;
+  try {
+    // Validate the format is yyyy-mm-dd
+    const dateFormatRegex = /^\d{4}-\d{2}-\d{2}$/;
+    if (!dateFormatRegex.test(skipUntilDate)) {
+      console.error(
+        "Invalid date format for VITE_SKIP_TAXES_TESTS_UNTIL. Expected format is 'yyyy-mm-dd'",
+      );
+      return false;
+    }
```

Comment:
Nit: Maybe this check is not needed given that we are passing the string to the Date object and checking if we created a valid object.

But I might not see the edge case here.

Reply:
It accepts a couple of different formats but I decided to make it explicit on "yyyy-mm-dd" to avoid confusion (epoch, iso timestamp, iso date, etc).